src/Microsoft.OData.Core/UriParser/SemanticAst/SelectExpandClauseExtensions.cs

```diff
@@ -86,7 +86,16 @@ internal static void GetSelectExpandPaths(this SelectExpandClause selectExpandCl
         /// <returns>String list generated from selected items</returns>
         internal static List<string> GetCurrentLevelSelectList(this SelectExpandClause selectExpandClause)

```

Comment:
>GetCurrentLevelSelectList [](start = 37, length = 25)

add unit test cases for test this static method works as expected.

Reply:
Resolved

Reply:
#Resolved